src/components/_classes/component/Component.js

```diff
@@ -2549,37 +2553,39 @@ export default class Component extends Element {
    * @return boolean - If component is valid or not.
    */
   checkData(data, flags, row) {
-    data = data || this.rootValue;
-    flags = flags || {};
-    row = row || this.data;
-    this.checkRefreshOn(flags.changed);
-    if (flags.noCheck) {
-      return true;
-    }
-    this.calculateComponentValue(data, flags, row);
-    this.checkComponentConditions(data, flags, row);
-    if (flags.noValidate) {
-      return true;
-    }
+      data = data || this.rootValue;
```

Comment:
This adds a space so that is why we see a diff here. Please remove the space so the commit is clean.